tutorials/shading/your_first_shader/index.rst

```diff
@@ -0,0 +1,27 @@
+Your first shader
+=================
+
```

Comment:
I didn't realize we could put text in the index.rst - this changes everything! :)